hs_core/tasks.py

```diff
@@ -539,3 +539,19 @@ def update_quota_usage_task(username):
     uq.update_used_value(used_val)
 
     return True
+
+
+@shared_task
+def update_web_services(services_url, api_token, timeout, res_id):
+    session = requests.Session()
+    session.headers.update(
+        {"Authorization": " ".join(("Token", str(api_token)))}
+    )
+    rest_url = str(services_url) + "/update-services/?res_id=" + str(res_id)
+    try:
+        response = session.post(rest_url, timeout=timeout)
+        logger.info(response)
+        return response
+    except (requests.exceptions.RequestException, ValueError) as e:
+        logger.error(e)
+        return e

```

Comment:
@kjlippold If there were exceptions when updating the service, the task would just log the exception/error and exit, which would leave geoserver not in sync with HydroShare. You may need to write a sync-up service to run on geoserver side on a regular basis, e.g., nightly or weekly, to call HydroShare REST API to make sure status of registered data on geoserver is in sync with HydroShare just to pick up these potential exception cases.